File: sica/modelos/Conexion.php
<?php
namespace modelos;

class Conexion{

    private $con;
    private $datos = array(
    "host" => "localhost",
    "user" => "root",
    "pass" => "",
    "db" => "sica"
  );

    public function __construct(){
    	$this->con = new \mysqli($this->datos['host'],
                               $this->datos['user'],
                               $this->datos['pass'],
                               $this->datos['db']);
    }

    public function consultaSimple($sql){
			$this->con->query($sql);
		}

	public function consultaRetorno($sql){
			$datos = $this->con->query($sql);
			return $datos;
	}

}

?>

File: requerimiento/src/modelos/Conexion.java
package modelos;

import java.sql.Connection;
import java.sql.DriverManager;

public class Conexion {
	

    public String db = "itinerario";
    public String url = "jdbc:mysql://localhost/"+db;
    public String user = "root";
    public String pass = "";
	
	private static Conexion con = new Conexion();
	
	private Conexion(){}
	
	public static Conexion getInstance(){
		return con;
	}
	
	public Connection conexion(){
		Connection cx = null;
		try{
			 Class.forName("com.mysql.jdbc.Driver");
             cx = DriverManager.getConnection(this.url, this.user, this.pass);
		}catch(Exception ex){
			 System.out.println("Error en la conexion "+ex.getLocalizedMessage());
		}
		return cx;
	}

}
